### PR TITLE
[14.0] purchase_sale_inter_company clean test and fix

### PR DESCRIPTION
**Pull request by sebastienbeau**
Clean test and fix broken feature

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@PierrickBrun @bealdav

File: purchase_sale_inter_company/models/purchase_order.py [hunk 1]
```diff
@@ -16,7 +16,10 @@ def button_approve(self, force=False):
         for purchase_order in self.sudo():
             # get the company from partner then trigger action of
             # intercompany relation
-            dest_company = purchase_order.partner_id.ref_company_ids
+            if purchase_order.partner_id.parent_id.is_company:
```
Comment (sebastienbeau) [inline purchase_sale_inter_company/models/purchase_order.py]: use "commercial_partner_id"

Comment (sebastienbeau) [reply purchase_sale_inter_company/models/purchase_order.py]: dest_company = purchase_order.partner_id.commercial_partner_id.ref_company_ids

Comment (Kev-Roche) [reply purchase_sale_inter_company/models/purchase_order.py]: Done

**Review by enriquemartin — APPROVED** [review]

**Review by Kev-Roche — APPROVED** [review]
LGTM

**Comment by elvise** [comment]
@sebastienbeau can you check why your merge was not done ?
Thanks

**Comment by elvise** [comment]
@sebastienbeau looks good now ?

**Pull request merged**